woocommerce/woocommerce-ios
# Settings > Switch Store: fix spinner color in the CTA in dark mode

Fixes #3439 

## Why

When dropping iOS 12 support in https://github.com/woocommerce/woocommerce-ios/commit/c274869417fb8f9f62a53010708dde15b708fc5c, we updated `FancyAnimatedButton`'s `UIActivityIndicatorView` style from `.gray` (deprecated in iOS 13) to `.medium`. Since the new style `.medium` just specifies the size, the previous gray color was missing and the new default color doesn't have enough contrast to white background in dark mode. From [Apple documentation](https://developer.apple.com/documentation/uikit/uiactivityindicatorview), we can just set the `color` property to specify the color:

> You can set the color of the activity indicator by using the color property.

In other `UIActivityIndicatorView` use cases that have similar style update, the background view has dark color in dark mode so we don't have to set the color manually in those cases.

## Changes

This PR set the `UIActivityIndicatorView`'s `color` to gray to maintain the same color as before.

## Testing

- Turn on dark mode for the simulator/device
- Log in to the app or stay logged in
- Go to settings
- Tap "Switch Store" --> the spinner in the "Continue" button at the bottom should be visible

## Example screenshots

before | after
-- | --
![Simulator Screen Shot - iPhone 11 - 2021-01-14 at 13 23 48](https://user-images.githubusercontent.com/1945542/104549071-1180a680-566d-11eb-8517-3f89bb406d82.png) | ![Simulator Screen Shot - iPhone 11 - 2021-01-14 at 13 23 54](https://user-images.githubusercontent.com/1945542/104549076-15142d80-566d-11eb-93cc-9061e3173627.png)
![Simulator Screen Shot - iPhone 11 - 2021-01-14 at 13 22 26](https://user-images.githubusercontent.com/1945542/104549089-1e9d9580-566d-11eb-95d3-048ba7893849.png) | ![Simulator Screen Shot - iPhone 11 - 2021-01-14 at 13 23 27](https://user-images.githubusercontent.com/1945542/104549095-22311c80-566d-11eb-8b9f-fcf767765ade.png)



Update release notes:
